cordova-js-src/exec.js

```diff
@@ -60,18 +60,32 @@ module.exports = function (success, fail, service, action, args) {
             // CB-5806 [Windows8] Add keepCallback support to proxy
             onSuccess = function (result, callbackOptions) {
                 callbackOptions = callbackOptions || {};
+                var callbackStatus;
+                if (callbackOptions.status !== null) {
```

Comment:
If the user passed undefined explicitly, it would have evaluated to falsy and in line 65 (pre-change) would have been replaced with cordova.callbackStatus.OK.  With your change, line 64's (post-change) test is true (undefined !== null), so local callbackStatus becomes undefined (line 65), and the status property is undefined (line 72 or 88).